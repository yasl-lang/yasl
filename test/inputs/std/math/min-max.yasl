echo math.max(1.0, 20, 4.0)
echo math.max(1, 2.0, 40)
echo math.max()
echo math.max(math.nan) === (math.nan)

echo math.min(1.0, 20, 4.0)
echo math.min(1, 2.0, 40)
echo math.min()
echo math.min(math.nan) === (math.nan)
