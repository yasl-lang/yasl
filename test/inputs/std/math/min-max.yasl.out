20
40
-inf
true
1.0
1
inf
true
